ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/ConnectExecutable.java

```diff
@@ -56,6 +56,7 @@ public static ConnectExecutable of(final String configFile) throws IOException {
   public void startAsync() {
     try {
       connect = connectDistributed.startConnect(workerProps);
+      Thread.currentThread().setContextClassLoader(this.getClass().getClassLoader());
```

Comment:
I would suggest to do the following way:
```
currentClassLoader = currentThread.getContextClassLoader()

try {
  connect = ...
} finally {
  currentThread.setContextClassLoader(currentClassLoader)
}
```

Also, add the ticket url for Connect team.

Reply:
Thanks for the feedback. Made the change!